packages/codegen-ui/lib/generate-form-definition/helpers/model-fields-configs.ts

```diff
@@ -65,6 +69,7 @@ export function getFieldConfigFromModelField({
       readOnly: field.readOnly,
       name: fieldName,
       value: 'true',
+      isArray: field.isArray,
```

Comment:
What is the reason for adding this? I think I'm missing where it's used.

Reply:
This will be used when the util function is generated. We will need to export our own ArrayField component that will exist in the utils file